src/composables/incomingTransactions.ts

```diff
@@ -0,0 +1,93 @@
+import { i18n } from '../store/plugins/languages';
+import { IAccount, ITransaction } from '../types';
+import { useNotifications } from './notifications';
+import { useGetter } from './vuex';
+import { useWebsocket } from './websocket';
+
+export function useIncomingTransactions() {
+  const { channelSubscribe } = useWebsocket();
+  const { addWalletNotification } = useNotifications();
+  const accounts = useGetter<IAccount[]>('accounts');
+  const getTxAmountTotal = useGetter<any>('getTxAmountTotal');
+  const getTxSymbol = useGetter<any>('getTxSymbol');
+
+  /**
+   * This function is used to format the given address.
+   * @param {string} address - the address to format.
+   * @returns {string} - returns the account's name if it exists,
+   * otherwise it returns the translated string "Account" + the account's index +1.
+  */
+  function formatAddress(address: string): string {
+    const account: IAccount | undefined = accounts.value.find(
```

Comment:
you no need ( Account | undefined ) ts automatically give that type

Reply:
![5090A497-0F41-4AC1-B9B3-7DCCDE63FC91](https://user-images.githubusercontent.com/10424827/212320190-a058c22d-867d-4e77-914c-749352fc73db.png)

Reply:
Also, there's a chance that the account cannot be found with `Array.find`.
and `.find` returns T | undefined


![DDAD6382-D845-4BC0-92D2-B6034ABFDB28](https://user-images.githubusercontent.com/10424827/212534601-0a8c3955-c00f-4b99-b854-1a7b0c5255d0.png)